modules/Core/includes/endpoints/SetGroupEndpoint.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/**
+ * @param int $id NamelessMC ID of user to view
+ * @param int $group_id ID of NamelessMC group
+ * 
+ * @return string JSON Array
+ */
+class SetGroupEndpoint extends EndpointBase {
+
+    public function __construct() {
+        $this->_route = 'setGroup';
+        $this->_module = 'Core';
+        $this->_description = 'Set a user\'s primary NamelessMC group';
+    }
+
+    public function execute(Nameless2API $api) {
```

Comment:
Another type

Reply:
As per our conversation on Discord, leaving this.